src/lib/layout/Sidebar.stories.ts

```diff
@@ -13,5 +15,20 @@ export const MobileSidebar = {
     viewport: {
       defaultViewport: 'mobile2'
     }
+  },
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+
+    await step('Open sidebar', async () => {
+      await userEvent.click(await canvas.findByRole('button'));
+      const menuOption = canvas.queryByText('Início');
+      await expect(menuOption).toBeTruthy();
```

Comment:
Maybe this would also need to be changed when we implement multiple language support.